jmix-ui/ui-data/src/main/java/io/jmix/uidata/filter/UiDataFilterSupport.java

```diff
@@ -117,6 +118,7 @@ public ScreenFragment createFilterConfigurationFragment(FrameOwner owner,
                 UiDataFilterConfigurationModelFragment.class);
 
         fragment.setDefaultForMeFieldVisible(defaultForMeFieldVisible);
+        fragment.setFilter(filter);

```

Comment:
`io.jmix.ui.component.Filter.Configuration#getOwner` should solve your task without changing public API